sympy/stats/frv.py

```diff
@@ -202,6 +202,11 @@ def pdf(self):
         return Lambda(x, Piecewise(*(
             [(v, Eq(k, x)) for k, v in self.dict.items()] + [(0, True)])))
 
+    @property
+    def characteristic_function(self):
+        t = Symbol('t')
```

Comment:
This should be a Dummy symbol (with real=True), as for other kinds of distributions. (The example in docstring also shows the inconsistency: sometimes _t, sometimes t. )